### PR TITLE
Fix spacing in issue navbar

### LINKED ISSUE #30226: New Label button align left but it should be right.
image

### PR DESCRIPTION
**Pull request by silverwind**
Create a new `issue-navbar` class specifically for this bar, previous class used in many places and I thought I had them all removed, but not this one.

Fixes: https://github.com/go-gitea/gitea/issues/30226

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: templates/repo/issue/labels.tmpl [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <div role="main" aria-label="{{.Title}}" class="page-content repository labels">
 	{{template "repo/header" .}}
 	<div class="ui container">
-		<div class="navbar tw-mb-4">
+		<div class="issue-navbar tw-mb-4">
```
Comment (wxiaoguang) [inline templates/repo/issue/labels.tmpl:5]: The code is similar, but why here has `mb-4`?

**Review by delvh — APPROVED** [review]

File: templates/repo/issue/labels.tmpl (hunk 1 above)
Comment (silverwind) [reply templates/repo/issue/labels.tmpl:5]: IIRC, the content below needs the spacing here. BTW there is another class `list-header` that serves a similar purpose than this one, I think they should be merged.

Comment (wxiaoguang) [reply templates/repo/issue/labels.tmpl:5]: I do not see so. 

Sometimes:

```
		<div class="issue-navbar">
			{{template "repo/issue/navbar" .}}
```

Sometimes:

```
		<div class="issue-navbar tw-mb-4">
			{{template "repo/issue/navbar" .}}
```

Comment (silverwind) [reply templates/repo/issue/labels.tmpl:5]: It does need it currently. On this page it looks like this without the margin:

image

Other pages have a divider below it to provide the spacing. Likely should find a better more general solution but this PR should be good enough on its own to fix the bug.

**Pull request merged**